README.md

```diff
@@ -1,21 +1,33 @@
-# Seq [![Join the chat at https://gitter.im/datalust/seq](https://img.shields.io/gitter/room/datalust/seq.svg)](https://gitter.im/datalust/seq) [![Feature Requests](https://img.shields.io/badge/features-uservoice-orange.svg)](https://seq.uservoice.com) [![Read Documentation](https://img.shields.io/badge/docs-online-blue.svg)](http://docs.getseq.net)
+# Seq [![Join the chat at https://gitter.im/datalust/seq](https://img.shields.io/gitter/room/datalust/seq.svg)](https://gitter.im/datalust/seq) [![Read Documentation](https://img.shields.io/badge/docs-online-blue.svg)](https://docs.getseq.net)
 
-Welcome to the issue tracker for [Seq](https://getseq.net)! You can **[search existing issues](https://github.com/datalust/seq-releases/issues)** or **[report an issue](https://github.com/datalust/seq-releases/issues/new)** here.
+Welcome to the hub for issues, design discussions and the feature roadmap for [Seq](https://getseq.net).
 
 ### What is Seq?
 
 Seq is a log server designed to speed up diagnostics in complex, asynchronous and distrubuted applications. It has a strong focus on _structured logging_, the idea that log events should carry important information in first-class properties.
 
 ### Bugs and feature requests
 
-If you think you may have found a bug in Seq, this is the place to report it.
+If you think you may have found a bug in Seq, this is the place to report it. You can **[search existing issues](https://github.com/datalust/seq-releases/issues)** or **[raise a new issue](https://github.com/datalust/seq-releases/issues/new)** here.
```

Comment:
with the repo being renamed, the links should point to `https://github.com/datalust/seq-tickets` instead of `https://github.com/datalust/seq-releases`

Reply:
Nice spotting, fixed, thanks 👍